docs/examples/03_backends_ros/06_ros_in_grasshopper.rst

```diff
@@ -0,0 +1,24 @@
+*******************************************************************************
```

Comment:
I deleted the `full_examples` page because the integration tests were running them against the wrong robot (they were created for the `Panda`, not the `UR5`). Surprisingly, they worked (because we don't check output on those integration tests), but it was very confusing and since the loading of robots (compared to the time in which the first examples were written), has now moved totally to the ros loader -which was partially the goal of the full examples-, I simply moved that to the normal examples and kept only one set of them using UR5. And so I had to put the Grasshopper example in a page of its own.